README.md

```diff
@@ -74,6 +74,44 @@ Then update the `package.json` file in your project with the new version:
   }
 ```
 
+### Working with the Pattern Library
+
+Employer Style Base includes a [pattern library](http://cb-talent-development.github.io/employer-style-base/) built with [Fractal](http://fractal.build/). The pattern library serves as a quick reference to the components that come with style base and how to use them.
+
+If you haven't already, from within the employer-style-base repo run
+
+```
+yarn install
+```
+
+This will install Fractal and Fractal CLI.
+
+In order to show styled components, you must first compile from `.scss` to `.css`. Run
+
+```
+npm dev
+```
+
+to both compile and watch the `.scss` files. If you've already started the Fractal server you may need to restart it.
+
+
+Now start the pattern library locally with
+
+```
+fractal start --sync
```

Comment:
I can't run this command without specifying the fully-qualified path of the executable - it may be that Fractal has some issues with Yarn. I'll play around with it tonight. Either way, we should have this bit of documentation refer to the `script` you created in `package.json` - `yarn run fractal`